### PR TITLE
Upgrade Radix and DS to latest

### PR DESCRIPTION
**Pull request by andy-hook**
Builds on https://github.com/radix-ui/website/pull/233

Upgrades Radix, DS to latest and applies relevant code changes.

### CODE REVIEW

**Review by peduarte — APPROVED** [review]
Nice one! Looking good

File: components/Highlights.tsx [hunk 1]
```diff
@@ -65,8 +65,8 @@ export function Highlights({ features }) {
       </Box>
 
       <Box css={{ width: 'fit-content' }} as="nav" aria-labelledby="site-component-info-heading">
-        <VisuallyHidden as="h2" id="site-component-info-heading">
-          Component Reference Links
+        <VisuallyHidden asChild id="site-component-info-heading">
+          <h2>Component Reference Links</h2>
```
Comment (benoitgrelard) [inline components/Highlights.tsx:69]: I know this works technically but I feel like DOM attributes sit better on the element itself then

```suggestion
        <VisuallyHidden asChild>
          <h2 id="site-component-info-heading">Component Reference Links</h2>
```

File: data/primitives/components/toolbar/0.1.0.mdx [hunk 2]
```diff
@@ -312,7 +312,7 @@ Used to visually separate items in the toolbar.
 
 ### Use with other primitives
 
-All our primitives which expose a `Trigger` part, such as `Dialog`, `AlertDialog`, `Popover`, `DropdownMenu` can be composed within a toolbar by using the `as` prop.
+All our primitives which expose a `Trigger` part, such as `Dialog`, `AlertDialog`, `Popover`, `DropdownMenu` can be composed within a toolbar by using the `asChild` prop.
```
Comment (benoitgrelard) [inline data/primitives/components/toolbar/0.1.0.mdx:315]: Good catch on this file!!

File: data/primitives/components/tooltip/0.1.0.mdx [hunk 3]
```diff
@@ -302,10 +302,12 @@ import * as Tooltip from '@radix-ui/react-tooltip';
 
 export default () => (
   <Tooltip.Root>
-    <Tooltip.Trigger as="__span__">
-      <button disabled style={{ __pointerEvents__: 'none' }}>
-        …
-      </button>
+    <Tooltip.Trigger asChild>
```
Comment (benoitgrelard) [inline data/primitives/components/tooltip/0.1.0.mdx:305]: does `asChild` and `<span>` need highlighting like `__span__` was before?
Not sure what's possible here @peduarte ?

File: data/primitives/components/tooltip/0.1.0.mdx [hunk 4]
```diff
@@ -418,13 +420,12 @@ export default () => (
 
 #### Implementation
 
-Use the [`Slot` utility](/docs/primitives/utilities/slot) to convert the trigger part into a slottable area. It will replace the trigger with the child that gets passed to it.
+Use the `asChild` prop to convert the trigger part into a slottable area. It will replace the trigger with the child that gets passed to it.
```
Comment (benoitgrelard) [inline data/primitives/components/tooltip/0.1.0.mdx:423]: Great catch again! Should we link `asChild` to its section in the docs?

File: data/primitives/components/toolbar/0.1.0.mdx (hunk 2 above)
Comment (benoitgrelard) [reply data/primitives/components/toolbar/0.1.0.mdx:315]: Should we link `asChild` to its section in the docs?

**Review by benoitgrelard — APPROVED** [review]

**Comment by andy-hook** [comment]
Solid recs @benoitgrelard, I've tweaked based on those.

**Comment by peduarte** [comment]
@andy-hook @benoitgrelard @jjenzz I hijacked this PR to update a pre-release of our DS, which in turn contains the latest primitives.

**Review by jjenzz — APPROVED** [review]

**Pull request merged**